transport/http2_client.go

```diff
@@ -236,9 +236,9 @@ func (t *http2Client) NewStream(ctx context.Context, callHdr *CallHdr) (_ *Strea
 	var timeout time.Duration
```

Comment:
It is used further below where it is propagated in a header.
On Apr 13, 2016 12:49, "Xiang Li" <notifications@github.com> wrote:

> In transport/http2_client.go
> <https://github.com/grpc/grpc-go/pull/634#discussion_r59583363>:
>
> > @@ -236,9 +236,9 @@ func (t *http2Client) NewStream(ctx context.Context, callHdr *CallHdr) (_ *Strea
> >  	var timeout time.Duration
>
> it seems like the whole timeout var can be removed.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/634/files/025674fec58a059fa5bdb94d3cd64c2b1c3242e4#r59583363>
>